core-domain/src/main/java/org/uniprot/core/impl/CrossReferenceImpl.java

```diff
@@ -5,34 +5,34 @@
 import java.util.List;
 import java.util.Objects;
 
-import org.uniprot.core.DBCrossReference;
-import org.uniprot.core.DatabaseType;
+import org.uniprot.core.CrossReference;
+import org.uniprot.core.Database;
 import org.uniprot.core.Property;
 import org.uniprot.core.util.Utils;
 
-public class DBCrossReferenceImpl<T extends DatabaseType> implements DBCrossReference<T> {
+public class CrossReferenceImpl<T extends Database> implements CrossReference<T> {
     private static final long serialVersionUID = 4318477387676269483L;
     protected T databaseType;
```

Comment:
Maybe the property name should be renamed to database.. What do you think? this will change the property name for the json response as well, because the method name was changed... just the property name that has not being changed...